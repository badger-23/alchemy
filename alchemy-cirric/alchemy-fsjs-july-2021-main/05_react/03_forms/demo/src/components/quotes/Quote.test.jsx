import React from 'react';
import { render, screen } from '@testing-library/react';
import Quote from './Quote';

describe('Quote', () => {
  it('displays a quote', () => {
    render(
      <Quote name="spot" quote="woof woof" image="example.com/image.png" />
    );

    const quote = screen.getByText('woof woof - spot');
    expect(quote).toMatchSnapshot();
  });
});
