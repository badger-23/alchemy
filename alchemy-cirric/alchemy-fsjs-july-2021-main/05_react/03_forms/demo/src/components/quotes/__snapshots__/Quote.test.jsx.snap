// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quote displays a quote 1`] = `
<figcaption>
  woof woof
   - 
  spot
</figcaption>
`;
